tests/foreman/cli/test_import.py

```diff
@@ -273,6 +273,62 @@ def test_reimport_orgs_default_negative(self, test_data):
             'rm -rf ${{HOME}}/.transition_data {}'.format(files['users'])
         )
 
+    @data(*import_org_data())
+    def test_import_orgs_recovery(self, test_data):
+        """@test: Try to Import organizations with the same name to invoke
+        usage of a recovery strategy (rename, map, none)
+
+        @feature: Import Organizations Recover
+
+        @assert: 2nd Import will rename the new organizations, 2nd import will
+        map them and the 3rd one will resul in No Action Taken
+
+        """
+        # prepare the data
+        files = dict(self.default_dataset[1])
+        new_dataset = update_csv_values(files['users'], test_data)
+        ssh.upload_file(new_dataset, new_dataset)
+        files['users'] = new_dataset
```

Comment:
I don't see the need of creating the new_dataset var then. That line could be something like `files['users'] = update_csv_values(files['users'], test_data)` because the new_dataset variable makes me think that is something new and then you are updating the previous value, because this the example I gave the code tell me more that in the end you want to update the value of that key.